src/reducers/MapReducer.js

```diff
@@ -157,6 +191,17 @@ export default (state = INITIAL_STATE, action) => {
 
             return {...state, filterCriteriaKeys: updatedFilterKeys, displaySites: newlyFilteredSites};
 
+        case FILTER_CRITERIA_RESET:
+            const filterResetSites = filterSites(state, INITIAL_STATE.filterCriteriaKeys);
+
+            return {...state, filterCriteriaKeys: INITIAL_STATE.filterCriteriaKeys, filterResultsScrutinyLoose: INITIAL_STATE.filterResultsScrutinyLoose, displaySites: filterResetSites};
+
+        case FILTER_TOGGLE_LOGIC_UPDATED:
+            const {filterToggleKey} = payload;
+            const updatedFilterResultsScrutinyLooseObject = updateFilterResultsScrutiny(state, filterToggleKey);
+            const newlyFiteredSitesWithNewToggleLogic = filterSites({sites: state.sites, filterResultsScrutinyLoose: updatedFilterResultsScrutinyLooseObject}, state.filterCriteriaKeys);
+
+            return {...state, filterResultsScrutinyLoose: updatedFilterResultsScrutinyLooseObject, displaySites: newlyFiteredSitesWithNewToggleLogic};
```

Comment:
If we toggle the `Every`/`Any` switches, we should also run a new filter call to get the latest results.